Library/Config.ts

```diff
@@ -133,6 +138,16 @@ class Config {
 
         return iKey;
     }
+
+    /**
+    * Validate UUID Format
+    * Specs taken from https://tools.ietf.org/html/rfc4122
```

Comment:
Since we are following what Breeze is doing now, I suggest linking to that code snippet instead of the UUID Format doc.